modules/admin-ui-picard/app/src/components/events/partials/ModalTabsAndPages/NewMetadataExtendedPage.js

```diff
@@ -1,10 +1,59 @@
 import React from "react";
+import { useTranslation } from 'react-i18next';
+import { Field } from "formik";
+import RenderMultiField from '../../../shared/wizard/RenderMultiField';
+import RenderField from '../../../shared/wizard/RenderField';
+import WizardNavigationButtons from '../../../shared/wizard/WizardNavigationButtons';
+
+const NewMetadataExtendedPage = ({ previousPage, nextPage, formik, header, extendedMetadataFields }) => {
```

Comment:
yes, I hope it will be added in the future. I started to implement this page, however I couldn't find the correct endpoint that provides these fields (neither in the documentation nor in the old code). That means this page is paused till we gather information about the questions in confluence.

Reply:
maybe add todos at the places, where things have to be added when we have the information